src/js/view/preview.js

```diff
@@ -1,13 +1,39 @@
 define([
-    'utils/underscore'
-], function(_) {
+    'utils/underscore',
+    'utils/helpers'
+], function(_, util) {
 
     var Preview = function(_model) {
         this.model = _model;
 
-        this.model.on('change:playlistItem', this.loadImage, this);
+        this.model.on('change:playlistItem', onPlaylistItem, this);
     };
 
+    function loadOnComplete() {
+        if (this.model.get('state') === 'complete') {
+            this.loadImage(this.model, this.model.get('playlistItem'));
+        }
+    }
+
+    function onPlaylistItem(model, playlistItem) {
+        var audio = (this.model.mediaModel.get('mediaType') === 'audio');
```

Comment:
Has this been tested with a mixed playlist? I don't think 'mediaType' is set until after 'playlistItem' changes, so the value of 'mediaType' is not up-to-date with the current playlist item at this point.